<launch>
  <arg name="device_ip"/>
  <arg name="port_number"/>
  <arg name="udp_topic"/>
  <arg name="queue_size" default="1"/>

  <node pkg="raw_udp_receiver" name="udp_socket_listener" type="udp_receiver_node" output="screen" ns="udp">
    <param name="device_ip" value="$(arg device_ip)" />
    <param name="port_number" type="int" value="$(arg port_number)" />
    <param name="udp_topic" value="$(arg udp_topic)" />
    <param name="queue_size" value="$(arg queue_size) "/>
  </node>
</launch>
